app/services/container_topology_service.rb

```diff
@@ -80,10 +80,10 @@ def entity_display_type(entity)
           "Container" # the container entity itself
         end
       else
-        if name.eql? "Vm"
-          'VM'
+        if entity.kind_of?(Vm)
+          name.upcase # turn Vm to VM because it's an abbreviation
```

Comment:
@jrafanie nice :) But yes, it's indeed everywhere.. I'm thinking this could be another use for draper, but we should probably tackle this one switch at a time, not one model..